### PR TITLE
[Samples][Corebot-App-Insights] Add FlightBookingRecognizer class

### PR DESCRIPTION
**Pull request by matiasroldan6**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

**Review by FedericoBernal — APPROVED** [review]

File: samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    public FlightBookingRecognizer(Configuration configuration, BotTelemetryClient telemetryClient) {
+        Boolean luisIsConfigured = StringUtils.isNotBlank(configuration.getProperty("LuisAppId"))
+            && StringUtils.isNotBlank(configuration.getProperty("LuisAPIKey"))
+            && StringUtils.isNotBlank(configuration.getProperty("LuisAPIHostName"));
+
+        if (luisIsConfigured) {
+            LuisApplication luisApplication = new LuisApplication(
+                configuration.getProperty("LuisAppId"),
+                configuration.getProperty("LuisAPIKey"),
+                String.format("https://%s", configuration.getProperty("LuisAPIHostName"))
```
Comment (Batta32) [inline samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java:42]:
```suggestion
                "https://".concat(configuration.getProperty("LuisAPIHostName"))
```

This is just an improvement, it's format to use `format` in these scenarios

File: samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.corebot.app.insights;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.lang3.StringUtils;
```
Comment (Batta32) [inline samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java:9]: Is this ok?

**Review by Batta32 — CHANGES_REQUESTED** [review]

File: samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java (hunk 1 above)
Comment (matiasroldan6) [reply samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java:42]: ok, we should maybe change this in sample 13 later

File: samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java (hunk 2 above)
Comment (matiasroldan6) [reply samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java:9]: isn't that the import for StringUtils?

Comment (Batta32) [reply samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/FlightBookingRecognizer.java:9]: But you are accessing from the `applicationinsights` library, this is weird

**Review by Batta32 — APPROVED** [review]

**Pull request merged**